### DISCUSSION TITLE
problem to list a rentable nft

### DISCUSSION THREAD

**Discussion by StephanGerbeth** [ConsenSys-archive/truffle#5987] [General Q&A]
Hi,

i try to reproduce your rentable nft marketplace sample based on the celo blockchain.
https://trufflesuite.com/guides/nft-rental-marketplace-2/

But currently i have a problem to list a rentable nft on the celo testnet. 
Everytime i try to send the transaction, to list the nft, i resolve the following error:

`Error: execution reverted: Not enough ether for listing fee`

TestRepo: https://github.com/StephanGerbeth/rentable-nft-marketplace/
TestScript: https://github.com/StephanGerbeth/rentable-nft-marketplace/blob/main/src/index.mjs
TestWallet: https://alfajores.celoscan.io/address/0xb8fe3795f6b49f35f7779f559e7a0431679ce402

What I have already noticed is that the resolved msg.value is 0. But i defined it in the send call. :( 

Can you help me to resolve this problem or to show me, what i'm doing wrong at the last call inside the testScript?

Cheers
Stephan

**Comment by paulcli** [comment]
It seems like you are setting the value field in the transaction options, but it's not being used in the send() function. You should pass the options object to the send() function like this:
`const txReceipt = await tx2.send(options);`

Make sure to replace the last line of your code `(tx2.send(options);)` with the line above. This should resolve the issue with the msg.value being 0 and allow you to send the correct listing fee with the transaction.
